modules/apps/object/object-test/src/testFunctional/tests/Object.testcase

```diff
@@ -323,6 +323,43 @@ definition {
 			locator1 = "CreateObject#SELECT_OBJECT_FIELD_IN_TABLE");
 	}
 
+	@description = "LPS-162021 - Verify that the delete modal 'Delet' button is disabled by default"
```

Comment:
@JhossephG Here is the typo, it should be `Delete` not `Delet`

Reply:
My bad, I just copied and pasted the ticket description into the test and didn't see this typo. But I fixed it! :)